### PR TITLE
[Bulk] Validation - Missing descriptions

### PR DESCRIPTION
**Pull request by vatmicro**
This PR fixes 29 missing description issues

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/walkthrough-working-with-the-maskedtextbox-control.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Walkthrough: Working with the MaskedTextBox Control"
+description: Learn about and walk through the process of working with the MaskedTextBox control to alert users when the value they're attempting to commit is not valid for the type.
```
Comment (JustPies) [inline dotnet-desktop-guide/framework/winforms/controls/walkthrough-working-with-the-maskedtextbox-control.md:3]: This is over the 160 character max.

File: dotnet-desktop-guide/framework/winforms/how-to-display-dialog-boxes-for-windows-forms.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Display Dialog Boxes"
+description: Learn how to display dialog boxes in a similar way to how you display  other forms in Windows applications.
```
Comment (JustPies) [inline dotnet-desktop-guide/framework/winforms/how-to-display-dialog-boxes-for-windows-forms.md:3]: There is an extra space between "display" and "other."

**Comment by vatmicro** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/controls/webbrowser-security.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "WebBrowser Security"
+description: Learn about and how to use WebBrowser Security, which is designed to work in full trust only. HTML in the control can come from external Web servers.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/webbrowser-security.md:3]:
```suggestion
description: Learn about WebBrowser Security, which is designed to work in full trust only. HTML in the control can come from external Web servers. 
```

File: dotnet-desktop-guide/framework/winforms/event-handlers-overview-windows-forms.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Event Handlers Overview"
+description: Learn about and get an overview of event handlers that execute code within the event handlers when an event is raised.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/event-handlers-overview-windows-forms.md:3]:
```suggestion
description: Learn about the standard Windows Forms event handler format, including which parameters are passed to the handlers.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**